indigo_app/templates/place/works.html

```diff
@@ -22,7 +22,11 @@
     </div>
   {% endblock %}
 
-  <div class="card">
+<div class="row">
+  <div data-vue-component="TaxonomyTOC"></div>
```

Comment:
the direct children of `.row` must always be `.col` otherwise the padding breaks.

Reply:
changed :white_check_mark:

Reply:
It looks much better without messing with native padding or margins